packages/discord.js/src/structures/GuildOnboarding.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const { Collection } = require('@discordjs/collection');
+const Base = require('./Base');
+const { GuildOnboardingPrompt } = require('./GuildOnboardingPrompt');
+
+/**
+ * Represents the onboarding data of a guild.
+ * @extends {Base}
+ */
+class GuildOnboarding extends Base {
+  constructor(client, data) {
+    super(client);
+
+    const guild = this.guild;
+
+    /**
+     * The id of the guild this onboarding data is for
+     * @type {Snowflake}
+     */
+    this.guildId = data.guild_id;
```

Comment:
```suggestion
    /**
     * The id of the guild this onboarding data is for
     * @type {Snowflake}
     */
    this.guildId = data.guild_id;
    
    const guild = this.guild;
```

we need to assign `this.guildId` before we use the getter that uses that field to get the guild